plugin/api.go

```diff
@@ -190,6 +190,9 @@ type API interface {
 	// GetFileInfo gets a File Info for a specific fileId
 	GetFileInfo(fileId string) (*model.FileInfo, *model.AppError)
 
+	// GetFileLink gets the public link to a file by fileId.
```

Comment:
Can you document the minimum version required here?
```suggestion
	// GetFileLink gets the public link to a file by fileId.
	//
	// Minimum server version: 5.6
```